.firebaserc

```diff
@@ -1,5 +1,6 @@
 {
   "projects": {
-    "default": "mutualaid-757f6"
+    "default": "mutualaid-757f6",
+    "production": "resilienceapp-6a0dd"
```

Comment:
you can now use `firebase use <project>` to switch between the different databases and credentials